src/ContentHubClient.php

```diff
@@ -1334,8 +1334,7 @@ public function addFilterToWebhook($filter_id, $webhook_id) {
    * @throws \Exception
    */
   public function removeFilterFromWebhook($filter_id, $webhook_id) {
-    $options = ['body' => json_encode(['filter_id' => $filter_id])];
-    $response = $this->delete("settings/webhooks/$webhook_id/filters", $options);
+    $response = $this->delete("filters/$filter_id/subscriptions/$webhook_id");
```

Comment:
As per documentation here https://dev.acquia.com/api-documentation/acquia-content-hub-api#tag/filters/operation/PUT_v2-filters-uuid-subscriptions
`filters/$filter_id/subscriptions/$webhook_id` should be `filters/$filter_id/subscriptions/$subscriptionID`

OR is this a new endpoint which is not listed in documentation?

Reply:
I had a discussion with Neha regarding this, she said that in documentation `subscriptionID` and `webhook_id` are considered as same.